docs/sdk-and-tools/notifier.md

```diff
@@ -0,0 +1,248 @@
+---
+id: notifier
+title: Events notifier
+---
+
+## Overview
+
+A MultiversX observer node can push block events to a notifier service, which will process
+and forward the events to subscribers (via RabbitMQ). This way, one can subscribe to a rabbitMQ
+queue and receive block events, whenever a block is committed to the chain, instead of 
+polling an API frequently.
+
+The GitHub repository for the notifier service can be found here:
```

Comment:
The GitHub repository for the notifier service can be found [here](https://github.com/multiversx/mx-chain-notifier-go)

Reply:
done